### PR TITLE
fix(manager default): scylla now uses Focal Fossa as default

### PR DESCRIPTION
**Pull request by Orenef11**
base image, hence need to update the default manager agent
to the correct repo.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by fgelcer — APPROVED** [review]
LGTM

**Comment by roydahan** [comment]
DO NOT MERGE!
We haven't switch officially to ubuntu AMI.

File: defaults/test_default.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@ mgmt_port: 10090
 scylla_repo: ''
 scylla_repo_m: 'http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylla-2020.1.repo'
 scylla_mgmt_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/rpm/centos/scylladb-manager-2.2.repo'
+scylla_mgmt_agent_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/deb/unstable/focal/master/latest/scylla-manager-master/scylla-manager.list'
```
Comment (roydahan) [inline defaults/test_default.yaml:11]: Why is it master and not the latest release of 2.3?

Comment (roydahan) [reply defaults/test_default.yaml:11]: Please change.

**Comment by roydahan** [comment]
We decided to proceed with Ubuntu AMI on 2021.1.
So please fix this PR and I'll merge it.

**Pull request closed without merging**